code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -85,7 +85,7 @@
 
 	var/list/TLV = list( // Breathable air.
 		"pressure" = new/datum/tlv(HAZARD_LOW_PRESSURE, WARNING_LOW_PRESSURE, WARNING_HIGH_PRESSURE, HAZARD_HIGH_PRESSURE), // kPa. Values are min2, min1, max1, max2
-		"temperature" = new/datum/tlv(T0C, T0C+10, T0C+40, T0C+66),
+		"temperature" = new/datum/tlv(BODYTEMP_COLD_WARNING_1, BODYTEMP_COLD_WARNING_1+10, BODYTEMP_HEAT_WARNING_1-27, BODYTEMP_HEAT_WARNING_1),
```

Comment:
Undocced change reeeeeeeee

Reply:
I said removed the hardcoded values for the BODYTEMP icon warnings in several places.  I will specifically mention it in the PR but it's also apart of ad502a97d6bd7e4585aaa40342b3b2351e8c8288

Reply:
This is the edited version of the PR:

> These values are based on the BODYTEMP icon warnings that you get when a room is too hot or too cold. There is also some minor code improvements where some of the icon heat and cold warnings were hardcoded values. I made this a constant in the __DEFINES.atmospherics.dm file that you can see below. **These values have been applied to air alarms, thermomachines, gas meters, and species code.**

The bolded section is the new mention of the air alarms included.